src/base/PhpBlockView.php

```diff
@@ -26,6 +26,15 @@
  */
 class PhpBlockView extends View
 {
+    public function init()
+    {
+        parent::init();
+        
+        $this->on(self:: EVENT_AFTER_RENDER, function() {
```

Comment:
this won't work and should throw a parse error, i think we need a unit test here.

```
self:: EVENT_AFTER_RENDER
```
whitespace after ::

Or have you locally tested this?

Reply:
Hmm I would expect but it works... I have to make a fix in `registerToAppView` anyway. Merge the js assets together is not safe because the array keys used as position (head, body end, etc.)